app/src/main/java/com/theayushyadav11/MessEase/utils/Mess.kt

```diff
@@ -523,7 +528,7 @@ class Mess(context: Context) {
      * @param onResult-> A callback that returns two strings: version and URL.
      */
     fun getUpdates(onResult: (String, String) -> Unit) {
-        val s = get("update")
+        val s = get("update", "7:30")
```

Comment:
change it to previous version.
```suggestion
        val s = get("update")
```